mysql/tests/test_mysql.py

```diff
@@ -13,7 +13,7 @@
 from datadog_checks.mysql import MySql
 
 from . import common, tags, variables
-from .common import MYSQL_VERSION_PARSED
+from .common import MYSQL_REPLICATION, MYSQL_VERSION_PARSED
 
 
```

Comment:
Could we add in a test in an environment where group replication is not active that `_is_group_replication_active` returns false?